.gitignore

```diff
@@ -6,3 +6,4 @@ bin
 build
 .env
 READ_BEFORE_COMMIT.md
+.vscode
```

Comment:
nit: Add this to your global `.gitignore` rather than us including all the different IDE artifacts in each repo

Reply:
Makes sense, won't add (but also can't add to my global as I tend to commit these folders for other projects).